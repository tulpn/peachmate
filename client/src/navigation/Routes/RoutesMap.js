
import CalendarContainer from "../../container/Calendar/Calendar"
import SettingsContainer from "../../container/Settings/Settings"
import ConnectContainer from "../../container/Connect/Connect"
import DashboardContainer from "../../container/Dashboard/Dashboard"
import PostList from "../../container/Posts/List/List"
import PostSave from "../../container/Posts/Save/Save"

export default [
    // Dashboard
    {
        title: "Dashboard",
        path: "/",
        component: DashboardContainer,
        exactMatch: true
    },
    {
        title: "Dashboard",
        path: "/dashboard",
        component: DashboardContainer,
        exactMatch: true
    },
    // Settings
    {
        title: "Settings",
        path: "/settings",
        component: SettingsContainer,
        exactMatch: true
    },
    // Posts
    {
        title: "Posts",
        path: "/posts",
        component: PostList,
        exactMatch: true
    },
    {
        title: "Posts",
        path: "/posts/list",
        component: PostList,
        exactMatch: true
    },
    {
        title: "Save Post",
        path: "/posts/save/:id?",
        component: PostSave,
        exactMatch: true
    },
    // Calendar
    {
        title: "Calendar",
        path: "/calendar",
        component: CalendarContainer,
        exactMatch: true
    },
    // Connect
    {
        title: "Connect",
        path: "/connect",
        component: ConnectContainer,
        exactMatch: true
    },
]