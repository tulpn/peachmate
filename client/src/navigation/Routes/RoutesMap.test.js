import RoutesMap from "./RoutesMap"

test("correct routes are exported",  () => {
    expect(RoutesMap.map(({path, title}) => (
        `${title}: ${path}`
    ))).toMatchSnapshot()
})
