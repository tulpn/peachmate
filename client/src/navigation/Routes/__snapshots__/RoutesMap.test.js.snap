// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correct routes are exported 1`] = `
Array [
  "Dashboard: /",
  "Dashboard: /dashboard",
  "Settings: /settings",
  "Posts: /posts",
  "Posts: /posts/list",
  "Save Post: /posts/save",
  "Calendar: /calendar",
  "Connect: /connect",
]
`;
